src/cli.rs

```diff
@@ -73,13 +73,88 @@ Join the community at https://logg.ing/community.
         "#,
     subcommand_required = true,
 )]
-pub struct Cli {
+pub enum Cli {
+    #[clap(name = "storage")]
+    /// Storage options
+    Storage(StorageOptions),
+
+    /// Generate shell completions
+    #[clap(name = "completions")]
+    Completion(CommandCompletionOptions),
+}
+
+// taken generously from https://github.com/jj-vcs/jj/blob/be32d4e3efbb9a51deadcc63635a5fb1526d0d6c/cli/src/commands/util/completion.rs#L23C1-L47C36
+// Using an explicit `doc` attribute prevents rustfmt from mangling the list
+// formatting without disabling rustfmt for the entire struct.
+#[doc = r#"Print a command-line-completion script
+
+Apply it by running one of these:
+
+- Bash: `source <(pb util completion bash)`
+- Fish: `pb util completion fish | source`
+- Nushell:
+     ```nu
+     pb util completion nushell | save "completions-pb.nu"
```

Comment:
`pb` is a different cli tool, a client to the parseable(server) actually. 

Can we have a argument/flag similar to `--help` that when set will print the completions for the specific shell. i.e.
```sh
parseable --shell-completion-config nushell | save "parseable-completions.nu"
```

Reply:
This is also a good way to do this, rather than adding a command. I'll implement this.